class-tgm-plugin-activation.php

```diff
@@ -448,6 +451,11 @@ public function init() {
 			if ( true === $this->has_forced_deactivation ) {
 				add_action( 'switch_theme', array( $this, 'force_deactivation' ) );
 			}
+
+			// Display forced activation notice, if present.
+			if ( current_user_can( 'manage_options' ) && is_admin() ) {
```

Comment:
No need for the action if there are no force activated plugins
`if ( true === $this->has_forced_activation && ...`